src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,36 @@
-export const Tabs = () => {};
+import { Tab } from '../../Types/Tab';
+
+type Props = {
+  tabs: Tab[];
+  selectedTab: Tab;
+  onTabSelected: (tabId: string) => void;
+};
+
+export const Tabs: React.FC<Props> = ({ tabs, selectedTab, onTabSelected }) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map((tab: Tab) => (
+          <li
+            key={tab.id}
+            data-cy="Tab"
+            className={tab === selectedTab ? 'is-active' : ''}
+          >
+            <a
+              href={`#${tab.id}`}
+              data-cy="TabLink"
+              onClick={() => onTabSelected(tab.id)}
+              onKeyDown={() => onTabSelected(tab.id)}
```

Comment:
Do you really need it? It works with Enter by default, no need to add all the keys